nixos/modules/misc/ids.nix

```diff
@@ -297,6 +297,7 @@
       clickhouse = 278;
       rslsync = 279;
       minio = 280;
+      postage = 281;
```

Comment:
Because of predictability. I maintain multiple systems and I think it would help if the `/var/lib/postage` directory is always owned by the same uid/gid. It will also help with backups/restores or copies between systems if the directories have the same uid/gid.

What would be the advantage of a dynamic uid/gid?